synapse/rest/admin/experimental_features.py

```diff
@@ -13,30 +13,20 @@
 #  limitations under the License.
 
 
-from enum import Enum
 from http import HTTPStatus
 from typing import TYPE_CHECKING, Dict, Tuple
 
 from synapse.api.errors import SynapseError
 from synapse.http.servlet import RestServlet, parse_json_object_from_request
 from synapse.http.site import SynapseRequest
 from synapse.rest.admin import admin_patterns, assert_requester_is_admin
+from synapse.storage.databases.main.experimental_features import ExperimentalFeature
 from synapse.types import JsonDict, UserID
 
 if TYPE_CHECKING:
     from synapse.server import HomeServer
 
 
-class ExperimentalFeature(str, Enum):
```

Comment:
This was moved to the experimental features database class to avoid a circular import that occurred when using this type in the presence handler.